samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
 
 // Android Resolver Exclusions Start
 android {
+	namespace "com.unity3d.player"
```

Comment:
indent is a bit off

Reply:
shouldn't namespace reflect Datadog in the name btw?

Reply:
Yes, probably, though this was inserted by Unity so I should check that it's not a project setting somewhere.